test/k8s/suite_test.go

```diff
@@ -19,6 +20,7 @@ func TestE2E(t *testing.T) {
 }
 
 var cluster *K8sCluster
+var promClient *framework.PromClient
```

Comment:
Is it a stateful client? If so, do we need to close something? If it's stateless, can we avoid making it global and create it when it's needed?